gravitee-apim-portal-webui-next/src/app/api-details/api-tab-documentation/api-tab-documentation.component.html

```diff
@@ -15,10 +15,27 @@
     limitations under the License.
 
 -->
-<div class="api-tab-documentation__side-bar">
-  <app-page-tree [pages]="pageNodes" (openFile)="showPage($event)" />
-</div>
+@if (pageNodes$ | async; as nodes) {
+  @if (nodes.length) {
+    <div class="api-tab-documentation__side-bar">
+      <app-page-tree [initialSelectedPage]="page" [pages]="nodes" (openFile)="showPage($event)" />
+    </div>
 
-<div class="api-tab-documentation__page-content">
-  {{ selectedPage }}
-</div>
+    <div class="api-tab-documentation__page-content">
+      @if (selectedPageData$ | async; as selectedPageData) {
+        @if (selectedPageData.result) {
+          <app-page [page]="selectedPageData.result"></app-page>
+        }
+        @if (selectedPageData.error) {
+          <div>TODO: Implement error message</div>
```

Comment:
What's the idea for implementing error message? Can we just display the error?

Reply:
It will need to be in a separate PR. I spoke with Audrey and we will redirecting the user to a 404 page.https://www.figma.com/file/eAS6p0kITPnEI0VPbmATi3/Developer-Portal?type=design&node-id=3955-14608&mode=design&t=ckmG9hH9Ta9WOgQs-0